sklearn/metrics/_plot/calibration_curve.py

```diff
@@ -0,0 +1,193 @@
+from .base import _check_classifer_response_method
```

Comment:
It is private so we could correct the typo

```suggestion
from .base import _check_classifier_response_method
```

Reply:
Correct the typo in the function definition and everywhere it's used? If so maybe that's better done in a separate PR..?